x-pack/plugins/apm/scripts/precommit.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable no-console*/
+/* eslint-disable import/no-extraneous-dependencies*/
+
+const execa = require('execa');
+const Listr = require('listr');
+const { resolve } = require('path');
+
+const cwd = resolve(__dirname, '../../../..');
+
+const execaOpts = { cwd, stderr: 'pipe' };
+
+const tasks = new Listr(
+  [
+    {
+      title: 'Jest',
+      task: () =>
+        execa(
+          'node',
+          [
```

Comment:
Could we just do `require.resolve('jest/bin/jest')` here? Not sure what the jest script written here does other than run the CLI.

Reply:
Is that the same as running `yarn jest --config x-pack/plugins/apm/jest.config.js`? Because looks like the latter is a little slower to pick up files.